docs/src/content/docs/commands/BITCOUNT.md

```diff
@@ -7,115 +7,130 @@ The `BITCOUNT` command in DiceDB is used to count the number of set bits (i.e.,
+4. Count the number of bits set to 1 within the specified range or the entire string if no range is specified.
+5. Return the count as an integer.
 
-```plaintext
-26
-```
-
-### Example 2: Counting bits in a specified range
-
-```plaintext
-SET mykey "foobar"
-BITCOUNT mykey 1 3
-```
+## Errors
 
-`Output:`
+1. `Non-existent Key`:
 
-```plaintext
-10
-```
+   - Error Message: None (treats it as an empty string and returns 0)
+   - Occurs if the specified key does not exist
 
-In this example, the command counts the bits set to 1 in the bytes from position 1 to 3 (inclusive) of the string "foobar".
-
-## Error Handling
+   ```bash
+   127.0.0.1:7379> BITCOUNT non_existent_key
+   (integer) 0
+   ```
 
-### Common Errors
+2. `Wrong Type of Key`: 
 
-1. `Non-existent Key`: If the specified key does not exist, DiceDB treats it as an empty string and returns 0.
+   - Error Message: `(error) WRONGTYPE Operation against a key holding the wrong kind of value`
+   - Occurs if the key exists but does not hold a string value
 
-   ```plaintext
-   BITCOUNT non_existent_key
+   ```bash
+   127.0.0.1:7379> LPUSH mylist "element"
+   (integer) 1
+   127.0.0.1:7379> BITCOUNT mylist
+   (error) WRONGTYPE Operation against a key holding the wrong kind of value
    ```
 
-   `Output:`
-
-   ```plaintext
-   0
-   ```
+3. `Invalid Syntax / Range`:
 
-2. `Wrong Type of Key`: If the key exists but does not hold a string value, DiceDB will return an error.
+   - Error Message: `(error) ERR value is not an integer or out of range`
+   - Occurs if the `start` or `end` parameters are not integers
 
-   ```plaintext
-   LPUSH mylist "element"
-   BITCOUNT mylist
+   ```bash
+   127.0.0.1:7379> SET mykey "foobar"
+   OK
+   127.0.0.1:7379> BITCOUNT mykey start end
+   (error) ERR value is not an integer or out of range
    ```
 
-   `Output:`
-
-   ```plaintext
-   (error) WRONGTYPE Operation against a key holding the wrong kind of value
+   ```bash
+   127.0.0.1:7379> SET mykey "foobar"
+   OK
+   127.0.0.1:7379> BITCOUNT mykey BIT
+   (error) ERR value is not an integer or out of range
    ```
 
-3. `Invalid Range`: If the `start` or `end` parameters are not integers, DiceDB will return an error.
+   
+4. `Out of Range Indices`:
 
-   ```plaintext
-   SET mykey "foobar"
-   BITCOUNT mykey start end
-   ```
+   - Error Message: None (will be handled gracefully by considering the valid range within the string)
+   - Occurs if the `start` or `end` parameters are out of range of the string length
 
-   `Output:`
-
-   ```plaintext
-   (error) ERR value is not an integer or out of range
+   ```bash
+   127.0.0.1:7379> SET mykey "foobar"
+   OK
+   127.0.0.1:7379> BITCOUNT mykey 5 6
+   (integer) 4
+   127.0.0.1:7379> BITCOUNT mykey 5 10
+   (integer) 4
    ```
 
-4. `Out of Range Indices`: If the `start` or `end` parameters are out of the range of the string length, DiceDB will handle it gracefully by considering the valid range within the string.
+## Examples
```

Comment:
nit:
```suggestion
## Example Usage
```

Reply:
I think in the original issue for this task/PR mentions the title to be "Examples": [https://github.com/DiceDB/dice/issues/838](https://github.com/DiceDB/dice/issues/838).

> Note: The title of the sections should be the exact strings mentioned above, like "Examples", "Return values", etc.

So can you confirm which one should we adhere to? Thanks!

Reply:
Please take a look at this as an example. https://dicedb.io/commands/set/. I understand there is inconsistency but let's try to make it similar to other pages. Thanks.

Reply:
Ok, done!